fe/fe-core/src/main/java/org/apache/doris/clone/LoadStatisticForTag.java

```diff
@@ -389,6 +389,16 @@ public void getBackendStatisticByClass(
 
         for (BackendLoadStatistic beStat : beLoadStatistics) {
             Classification clazz = beStat.getClazz(medium);
+            long debugHighBeId = DebugPointUtil.getDebugParamOrDefault("FE.HIGH_LOAD_BE_ID", -1L);
```

Comment:
no need 392 ~ 401  ？LoadStatisticForTag.init has set it.

Reply:
removed